Removed user id and password from login response

bisonlou opened the pull request:
### What does this PR do
- Removed the user Id from the login response
- Removed the password from the login response
- Formated user_model file to PEP8 standard-

bisonlou committed:
Removed user id and password from login response

bisonlou commented:
I also changed the phone number expected datatype from int to string @kalsmic

PatrickMugayaJoel reviewed (commented):
Looks good. We shall get there one step at a time.
With time the 'debug=true' in 'run_server.py' file will go to the config.py file or be set in environment

PatrickMugayaJoel committed:
Create .flaskenv

This file will have environment variables that should not change for all users or even when deployed for production. These variables are not private.

PatrickMugayaJoel committed:
Update .gitignore

I made an assumption we are all going to call our virtual environments venv and not myenv

PatrickMugayaJoel committed:
Update __init__.py

- I moved database functionality out of this file to a database module.
- I added configurations from config.py into flask's app.config
- I tweaked the logs appearance
- I added an index route
- Imported user_routes and incident_routes as opposed to running them as functions. (Requires peer review)

PatrickMugayaJoel committed:
Update config.py

- Removed default values. Thus the env var should always be there in the environment.
- Removed development and production classes. The "FLASK_ENV" environment variable will serve their purpose.
- Left a commented testing class to show how we would have handled different config classes.

PatrickMugayaJoel committed:
Create connection.py

PatrickMugayaJoel committed:
Update incident_model.py

PatrickMugayaJoel committed:
Update user_model.py

PatrickMugayaJoel committed:
Update incident_routes.py

I got rid of classes and methods. (requires peer review)

PatrickMugayaJoel committed:
Update user_routes.py

PatrickMugayaJoel committed:
Update helper.py

PatrickMugayaJoel committed:
Update incident_veiws.py

PatrickMugayaJoel committed:
Update user_veiws.py

PatrickMugayaJoel committed:
deleteing virtual environment

PatrickMugayaJoel committed:
Update README.md

PatrickMugayaJoel committed:
Update requirements.txt

PatrickMugayaJoel committed:
Update run_server.py

PatrickMugayaJoel committed:
Update base.py

PatrickMugayaJoel committed:
Update connection.py

PatrickMugayaJoel committed:
Update requirements.txt

bisonlou committed:
Removed Passlib and instead used weerkzeug for hashing

bisonlou committed:
Fixed confilcts

bisonlou committed:
removed flaskenv

kalsmic commented:
Well done @bisonlou, looking forward to the tests

bisonlou committed:
removed passlib

bisonlou committed:
Fixed merge confilicts

bisonlou committed:
fixed conflicts

bisonlou committed:
Added succesful login and register tests

bisonlou commented:
Could only add succesful login and register tests because ther is no validation at the moment

kalsmic commented on app/views/incident_veiws.py:
```diff
@@ -8,7 +8,7 @@
 class CreateIncident(MethodView):
     @token_required
-    def post(current_user, self):
+    def post(self, current_user):
```
👍

kalsmic reviewed (approved):
Thanks, @bisonlou, You have covered the tasks as laid out in the user story.
